src/PJ_helmert.c

```diff
@@ -486,6 +486,22 @@ PJ *TRANSFORMATION(helmert, 0) {
         P->right = PJ_IO_UNITS_PROJECTED;
     }
 
+    /* Support the classic PROJ towgs84 parameter, but allow later overrides.*/
+    /* Note that if towgs84 is specified, the datum_params array is set up   */
+    /* for us automagically by the pj_datum_set call in pj_init_ctx */
+    if (pj_param (P->ctx, P->params, "ttowgs84").i) {
```

Comment:
pj_param_exists?

Reply:
Yes, it makes sense here, and we could redesign the entire parameter handling in this function to  use `pj_param_exists`, which I think we should do, but not now. Changed this instance, because it is different in havign its work already done by a friend up in `pj_datum_set`